src/main/java/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/bitbucket/Pullrequest.java

```diff
@@ -214,8 +221,13 @@ public String getContent() {
             return content;
         }
 
-        public void setContent(String content) {
-            this.content = content;
+        public void setContent(Object content) {
```

Comment:
The content field of the 1.0 and 2.0 APIs are different. In 2.0 it's an object. This change makes it so that content isn't a proper bean property (different type for the getter and the setter), which isn't ideal, but works for the usage of this class within the plugin. The alternative is to create a parallel V2 comment class that knows how to parse that structure instead and change the calling code to use that new class when retrieving the comments.